docker/addons/docker-compose.dex.yaml

```diff
@@ -57,9 +57,19 @@ services:
       INDEXER_CHRONOS_LEADERBOARD_CROSS_MARKET_IDS: ${INDEXER_CHRONOS_LEADERBOARD_CROSS_MARKET_IDS}
       INDEXER_CHRONOS_LEADERBOARD_CUSTOM_RES: ${INDEXER_CHRONOS_LEADERBOARD_CUSTOM_RES}
       INDEXER_VOLUME_SINGLE_SIDE_HEIGHT: ${INDEXER_VOLUME_SINGLE_SIDE_HEIGHT}
+
+      INDEXER_HTTP_LISTEN_ADDR: 0.0.0.0:9910
+      INDEXER_GRPC_LISTEN_ADDR: 0.0.0.0:4444
+      INDEXER_HTTP_TLS_CERT: /apps/data/public.crt
+      INDEXER_HTTP_TLS_KEY: /apps/data/private.key
+      INDEXER_HTTP_HORACLE_ADDR: ${INDEXER_HTTP_HORACLE_ADDR}
```

Comment:
here we can uniform with other vars:
INDEXER_HTTP_HORACLE_ADDR: https://k8s.${NETWORK}.horacle.injective.network/height/latest

Reply:
We can input whole string, it's more flexible I think